packages/yoroi-extension/app/components/common/Autocomplete.js

```diff
@@ -7,9 +7,9 @@ import { useCombobox, useMultipleSelection } from 'downshift';
 import { Input, Box, InputLabel, FormControl, FormHelperText, Chip, useTheme } from '@mui/material';
 import { styled } from '@mui/system';
 import { slice } from 'lodash';
-import SuccessIcon from '../../assets/images/forms/done.inline.svg';
-import ErrorIcon from '../../assets/images/forms/error.inline.svg';
-import CloseIcon from '../../assets/images/close-chip.inline.svg';
+import { ReactComponent as SuccessIcon}  from '../../assets/images/forms/done.inline.svg';
```

Comment:
Please add one extra space after the closing bracket ` }` and this applies for every import statment!

Reply:
Fixed.